packages/components/types/overrideClassName.ts

```diff
@@ -0,0 +1,3 @@
+export type OverrideClassName<T> = Omit<T, "className"> & {
+  classNameOverride?: string
+}
```

Comment:
You removed my extra type safety :')

```suggestion
export type OverrideClassName<T extends Record<string, any>> =
  | Omit<T, "className"> & { classNameOverride?: string }
```